### ISSUE TITLE
The reproduced model fails to achieve the accuracy reported in the paper

### ISSUE THREAD

**Issue by FogSue** [jiawen-zhu/ViPT#26]
Thank you for your open-source code. I noticed that the model supports two types of backbones: vit_ce and vit. So, I downloaded the models provided by ostrack, vitb_256_mae_ce_32x4_ep300 and vitb_256_mae_32x4_ep300, and placed them in the ./pretrained directory (OSTrack_ce_ep0300.pth.tar and OSTrack_ep0300.pth.tar). By modifying the YAML file, I attempted four combinations of pre-trained weights and backbones in Lasher：
 OSTrack_ep0300.pth+vit_base_patch16_224_prompt
 
[vipt-deep_rgbt_os300+vit.log](https://github.com/jiawen-zhu/ViPT/files/14816381/vipt-deep_rgbt_os300%2Bvit.log)

 OSTrack_ep0300.pth+vit_base_patch16_224_ce_prompt
 
[vipt-deep_rgbt_os300+vitce.log](https://github.com/jiawen-zhu/ViPT/files/14816383/vipt-deep_rgbt_os300%2Bvitce.log)

 OSTrack_ce_ep0300.pth+vit_base_patch16_224_ce_prompt
 
[vipt-deep_rgbt_os300ce+vitce.log](https://github.com/jiawen-zhu/ViPT/files/14816386/vipt-deep_rgbt_os300ce%2Bvitce.log)

 OSTrack_ce_ep0300.pth+vit_base_patch16_224_prompt
 
[vipt-deep_rgbt_os300ce+vit.log](https://github.com/jiawen-zhu/ViPT/files/14816388/vipt-deep_rgbt_os300ce%2Bvit.log)

However, none of these combinations achieved the accuracy reported in the vipt-deep_rgbt.log that you provided. Could you please advise on what might be the issue with my reproduction?(4090*2)
By the way, in order to resolve the issue of not finding images during the reading process, I made the following changes.
![微信图片_20240331224006](https://github.com/jiawen-zhu/ViPT/assets/79622142/dc30ef9a-1a20-496d-9bf4-b5ea8d32d8cc)

**Comment by aqizhoua** [comment]
佬，请问你后续有复现出类似的结果吗
